howso/feature_conviction.amlg

```diff
@@ -53,34 +53,32 @@
 	)
 
 	;computes the conviction for each feature and returns an assoc of feature -> conviction value
-	;
-	;parameters:
-	; features: list of label names
-	; action_features: optional, list of action features to use as the baseline for conviction instead of the full model
-	; case_ids: optional list of cases for which to use as the model to compute feature conviction
-	; familiarity_conviction_addition: deafult to true, calculate and output familiarity conviction of adding the specified features
-	; familiarity_conviction_removal: default to false, calculate and output familiarity conviction of removing the specified features
-	; use_case_weights: flag, if set to true will scale influence weights by each case's weight_feature weight. If unspecified,
-	;   			case weights will be used if the trainee has them.
-	; weight_feature: optional, default '.case_weight'.  name of feature whose values to use as case weights
 	#get_feature_conviction
 	(declare
 		(assoc
 			;{type "list" values "string"}
+			;list of label names
```

Comment:
wow this is a really old description, please edit this one to "list of feature names"

Reply:
Addressed in: [d54a657](https://github.com/howsoai/howso-engine/pull/239/commits/d54a6575c30a5c75415b2cc52e96d51cfb0406c6)